src/app/models/user.server.model.ts

```diff
@@ -16,20 +17,19 @@ const compileUserModel = (db: Mongoose) => {
     email: {
       type: String,
       trim: true,
-      unique: 'Account already exists with this email',
+      unique: true,
       required: 'Please enter your email',
       validate: {
         // Check if email entered exists in the Agency collection
-        validator: async (value: string) => {
-          return new Promise<boolean>((resolve, reject) => {
+        validator: (value: string) => {
+          return new Promise<boolean>((resolve) => {
```

Comment:
nit but can we just make this validator async? callbacks are so last decade

Reply:
Done :P